### ISSUE TITLE
Configuration of cassandra should be modified prior to entrypoint script

### ISSUE THREAD

**Issue by VadimKirilchuk** [spotify/docker-cassandra#12]
The issue is that if I'd make an image from spotify/cassandra-singlenode with modified configuration I had to modify script file which runs in entrypoint. Also the changes would be applied only when entrypoint  is executed.

It looks confusing. Why singlenode configuration can't be set in image itself, i.e.
RUN sed -i .... && sed -i ...

So, entrypoint could be just a simple command like "cassandra -f".

Wdyt?

**Comment by saidbouras** [comment]
Yes, it's less confusing to configure when building the image, i use this image for all my integration tests with cassandra and it works fine. 
https://github.com/saidbouras/cassandra-docker-unit
